semantic-conventions/src/opentelemetry/semconv/main.py

```diff
@@ -214,6 +215,14 @@ def add_md_parser(subparsers):
         dest="md_enable_deprecated",
         action="store_false",
     )
+    parser.add_argument(
+        "--specification-version",
+        help="Set specification version to use.",
```

Comment:
To manage expectations, maybe mention that this does not yet support managing arbitrary internal links (although I guess this might then be forgotten even if we implement that)

Reply:
I clarified the comment.